src/js/utils/list-utils.jsx

```diff
@@ -35,3 +36,20 @@ export const findActions = (actionList, row, props) => {
   return filteredByMinimumRequiredRole;
 };
 
+export const transformFilterParams = (filterValues, filterAccessors) => Object.keys(filterValues)
+  .filter(key => (filterAccessors[key] && !!filterValues[key]))
+  .reduce((acc, key) => {
+    const { name, accessor } = filterAccessors[key];
+    const theValue = filterValues[name];
+
+    let value = '';
+    if (!accessor) {
+      value = theValue;
+    } else if (Array.isArray(theValue)) {
+      value = _.map(theValue, accessor);
+    } else {
+      value = _.get(theValue, accessor);
+    }
+    return { ...acc, [key]: value };
+  }, {});
```

Comment:
It's my "personal" reflection, because I hate using `else` and `let` :D
if you don't mind, you could do it like this in more functional way:
```
if (!accessor) {
  return { ...acc, [key]: theValue };
}
if (Array.isArray(theValue)) {
  return { ...acc, [key]: _.map(theValue, accessor) };
}
return { ...acc, [key]: _.get(theValue, accessor) };
```